### PR TITLE
Allow values lower than 3 for skip hours

### PR DESCRIPTION
**Pull request by scorillo**
Change the skip hour value in order to catch a bug in /types/hour.pp

### CODE REVIEW

File: types/hour.pp [hunk 1]
```diff
@@ -2,8 +2,8 @@
 type Cron::Hour = Variant[
   Integer[0,23],
   Pattern[/(?x)\A(
-    \* ( \/ ( 1?[0-9]|2[0-3]|[3-9] ) )?
-    |       ( 1?[0-9]|2[0-3] ) ( - ( 1?[0-9]|2[0-3] ) ( \/ ( 1[0-9]|2[0-3]|[3-9] ) )? )?
-        ( , ( 1?[0-9]|2[0-3] ) ( - ( 1?[0-9]|2[0-3] ) ( \/ ( 1[0-9]|2[0-3]|[3-9] ) )? )? )*
+    \* ( \/ ( 1[0-9]|2[0-3]|[1-9] ) )?
+    |       ( 1?[0-9]|2[0-3] ) ( - ( 1?[0-9]|2[0-3] ) ( \/ ( 1[0-9]|2[0-3]|[1-9] ) )? )?
+        ( , ( 1?[0-9]|2[0-3] ) ( - ( 1?[0-9]|2[0-3] ) ( \/ ( 1[0-9]|2[0-3]|[1-9] ) )? )? )*
```
Comment (smortex) [inline types/hour.pp:7]: :+1: for rejecting `*/0`.

On a side note, anything greater than `/12` does not really make sense is is likely an error.  Maybe we do not want to accept such values?

**Review by smortex — APPROVED** [review]

**Comment by scorillo** [comment]
@alexjfisher @smortex can this be merged?

File: types/hour.pp (hunk 1 above)
Comment (smortex) [inline types/hour.pp:7]:
```suggestion
    \* ( \/ ( 1[0-2]|[1-9] ) )?
    |       ( 1?[0-9]|2[0-3] ) ( - ( 1?[0-9]|2[0-3] ) ( \/ ( 1[0-2]|[1-9] ) )? )?
        ( , ( 1?[0-9]|2[0-3] ) ( - ( 1?[0-9]|2[0-3] ) ( \/ ( 1[0-2]|[1-9] ) )? )? )*
```

Comment (scorillo) [reply types/hour.pp:7]: Doesn't 1-19/15 mean 1 and 16, which are valid hours? :wink:

Comment (smortex) [reply types/hour.pp:7]: I expected `1-19/15` to mean `15` but you are right.

```
Following a range with ``/<number>'' specifies skips of the number's value through the range.
```

Forget this :upside_down_face:

**Comment by smortex** [comment]
Thanks!

**Pull request merged**